[2024-05-21T14:30:28.058+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-21T14:30:28.074+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v07.get_age scheduled__2021-12-21T00:00:00+00:00 [queued]>
[2024-05-21T14:30:28.081+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v07.get_age scheduled__2021-12-21T00:00:00+00:00 [queued]>
[2024-05-21T14:30:28.082+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-21T14:30:28.094+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_age> on 2021-12-21 00:00:00+00:00
[2024-05-21T14:30:28.099+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3010) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-21T14:30:28.101+0000] {standard_task_runner.py:63} INFO - Started process 3011 to run task
[2024-05-21T14:30:28.101+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_dag_with_python_operator_v07', 'get_age', 'scheduled__2021-12-21T00:00:00+00:00', '--job-id', '241', '--raw', '--subdir', 'DAGS_FOLDER/second_dag.py', '--cfg-path', '/tmp/tmp5xdcjg4u']
[2024-05-21T14:30:28.102+0000] {standard_task_runner.py:91} INFO - Job 241: Subtask get_age
[2024-05-21T14:30:28.139+0000] {task_command.py:426} INFO - Running <TaskInstance: our_dag_with_python_operator_v07.get_age scheduled__2021-12-21T00:00:00+00:00 [running]> on host c420abfdc210
[2024-05-21T14:30:28.212+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test' AIRFLOW_CTX_DAG_ID='our_dag_with_python_operator_v07' AIRFLOW_CTX_TASK_ID='get_age' AIRFLOW_CTX_EXECUTION_DATE='2021-12-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-12-21T00:00:00+00:00'
[2024-05-21T14:30:28.213+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-21T14:30:28.235+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-21T14:30:28.237+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-21T14:30:28.244+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_dag_with_python_operator_v07, task_id=get_age, run_id=scheduled__2021-12-21T00:00:00+00:00, execution_date=20211221T000000, start_date=20240521T143028, end_date=20240521T143028
[2024-05-21T14:30:28.275+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-21T14:30:28.292+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-21T14:30:28.296+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
